samples/README.md

```diff
@@ -1,12 +1,14 @@
-## Using basic AzureML APIs
+## Azure ML samples
+These samples are short code examples for using the Azure Machine Learning service with the R SDK. If you are new to the R SDK, we recommend that you first take a look at the more detailed end-to-end [vignettes](../vignettes).
 
-Before running the samples in RStudio, change to the sample directory in Rstudio using `setwd(dirname)`.
-The examples assume that the data and scripts are in the current directory.
+Before running a sample in RStudio, set the working directory to the folder that contains the sample script in RStudio using `setwd(dirname)` or Session -> Set Working Directory -> To Source File Location. Each vignette assumes that the data and scripts are in the current working directory.
 
-1. [setup.R](setup.R): Setup workspace before running samples.
-2. [Train on ML Compute](training/train-on-amlcompute): Train a model using an ML Compute cluster as compute target.
-3. [Deploying a model to Cloud](deployment/deploy-to-cloud): Deploy a model as a Webservice.
-4. [Deploying a model to Local](deployment/deploy-to-local): Deploy a model as a local Webservice.
+1. [train-on-amlcompute](training/train-on-amlcompute): Train a model on a remote AmlCompute cluster.
+2. [train-on-local](training/train-on-local): Train a model locally with Docker.
+2. [deploy-to-aci](deployment/deploy-to-aci): Deploy a model as a web service to Azure Container Instances (ACI).
+3. [deploy-to-local](deployment/deploy-to-local): Deploy a model as a web service locally.
+
+> Before you run these samples, make sure you have an Azure Machine Learning workspace. You can follow the [configuration vignette](../vignettes/configuration.Rmd) to set up a workspace. (You do not need to do this if you are running these examples on an Azure Machine Learning compute instance).
```

Comment:
Same here

Reply:
would we also need full URL for the numbered list where we link to each example?